docs/portal_config.md

```diff
@@ -718,6 +718,12 @@ Below is an example, with inline comments describing what each JSON block config
       "service": "query_page"
     }
   },
+  "userAccessToSite": { // optional: user must have access to a resorces to acess the site
```

Comment:
shoudl add something here to explain that this access restriction feature takes precedence over public pages

Reply:
I think for now this is okay. But it might be debateable in the future if a site has both this option enabled and public pages, which should takes precedence

Reply:
updated